addons/l10n_il/data/account_tax_template_data.xml

```diff
@@ -392,33 +393,34 @@
         <field name="description">0%</field>
         <field name="name">VAT Import Line</field>
         <field name="description">VAT Import Line</field>
-        <field name="amount">0</field>
-        <field name="amount_type">percent</field>
+        <field name="amount">100</field>
+        <field name="amount_type">division</field>
         <field name="type_tax_use">purchase</field>
         <field name="chart_template_id" ref="il_chart_template"/>
         <field name="tax_group_id" ref="tax_group_vat_17"/>
```

Comment:
@hedshefer The tax group here is weird but it was weird before.

Reply:
@jco-odoo Why is it weird?

Reply:
because it is not a tax of 17% and you see it in the invoice tax summary as being 17% while the entire line is taken.  (I would be surprised that all of your import taxes would be 17%) Why not have a separate tax group for it?

Reply:
@jco-odoo 
This is how we report it in the tax report. The entire line included in the 17% input, no metter if it's taken from the import line or from the normal 17% input tax.